releasenotes/notes/deprecate-process_config.enabled-89e9ab7b0193cc04.yaml

```diff
@@ -0,0 +1,12 @@
+---
+features:
+  - |
+    Configuration `process_config.enabled` is now split into two settings: `process_config.process_collection.enabled` and `process_config.container_collection.enabled`. This will allow better control over the process Agent.
+  - |
+    `process_config.enabled` now translates to these new settings:
+    * `process_config.enabled=true`: `process_config.process_collection.enabled=true`
+    * `process_config.enabled=false`: `process_config.container_collection.enabled=true` and `process_config.process_collection.enabled=false`
+    * `process_config.enabled=disabled`: `process_config.container_collection.enabled=false` and `process_config.process_collection.enabled=false`
+
```

Comment:
We should merge those to entries. Once we generate the CHANGELOG with `reno`, order is not guarantee. So we want everything in one entry for users.

Reply:
I didn't know that order is not guaranteed! I combined both into one entry